ext/kernel/operators.c

```diff
@@ -893,6 +894,42 @@ void zephir_ceil(zval *return_value, zval *op1 TSRMLS_DC)
 	RETURN_FALSE;
 }
 
+extern double _php_math_round(double value, int places, int mode);
+void zephir_round(zval *return_value, zval *op1, zval *op2, zval *op3 TSRMLS_DC)
+{
+    int places = 0;
+	long mode = PHP_ROUND_HALF_UP;
+    double return_val;
+    
+    convert_scalar_to_number_ex(&op1);
+
+    if (op2) {
+        places = zephir_get_intval_ex(op2);
+    }
+    if (op3) {
+        mode = zephir_get_intval_ex(op3);
+    }
+
+	switch (Z_TYPE_PP(&op1)) {
```

Comment:
Can you fix? I mean spaces
We are using tabs
https://github.com/phalcon/zephir/blob/master/.editorconfig#L15